sln/OData.Tests.Performance.sln

```diff
@@ -1,22 +1,22 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30225.117

```

Comment:
The changes here may have been inadvertent but we may want to retain Visual Studio 14 (VS 2015), or Visual Studio 15 (VS 2017) at least...